pynumdiff/tests/test_diff_methods.py

```diff
@@ -33,32 +35,32 @@ def iterated_fourth_order(*args, **kwargs): return fourth_order(*args, **kwargs)
 diff_methods_and_params = [
     (first_order, {}), (second_order, {}), (fourth_order, {}), # empty dictionary for the case of no parameters
     (iterated_second_order, {'num_iterations':5}), (iterated_fourth_order, {'num_iterations':10}),
-    (lineardiff, {'order':3, 'gamma':5, 'window_size':11, 'solver':'CLARABEL'}), (lineardiff, [3, 5, 11], {'solver':'CLARABEL'}),
-    (polydiff, {'degree':2, 'window_size':3}), (polydiff, [2, 3]),
-    (savgoldiff, {'degree':2, 'window_size':5, 'smoothing_win':5}), (savgoldiff, [2, 5, 5]),
-    (spectraldiff, {'high_freq_cutoff':0.2}), (spectraldiff, [0.2]),
-    (mediandiff, {'window_size':3, 'num_iterations':2}), (mediandiff, [3, 2], {'iterate':True}),
     (meandiff, {'window_size':3, 'num_iterations':2}), (meandiff, [3, 2], {'iterate':True}),
+    (mediandiff, {'window_size':3, 'num_iterations':2}), (mediandiff, [3, 2], {'iterate':True}),
     (gaussiandiff, {'window_size':5}), (gaussiandiff, [5]),
     (friedrichsdiff, {'window_size':5}), (friedrichsdiff, [5]),
     (butterdiff, {'filter_order':3, 'cutoff_freq':0.074}), (butterdiff, [3, 0.074]),
     (splinediff, {'degree':5, 's':2}), (splinediff, [5, 2]),
+    (spline_irreg_step, {'degree':5, 's':2}),
+    (polydiff, {'degree':2, 'window_size':3}), (polydiff, [2, 3]),
+    (savgoldiff, {'degree':2, 'window_size':5, 'smoothing_win':5}), (savgoldiff, [2, 5, 5]),
```

Comment:
I reordered these a bit. They have a canonical ordering in my mind now, and I'm slowly getting all the listings of them to agree.